project-3/jobs.cpp

```diff
@@ -1,24 +1,24 @@
 #include "jobs.hpp"
 
-//class for your basic Job; (copied from project2)
-Job::Job()
-{
-	arrivalTime = 0;
-	serviceTime = 0;
-	priority = -1;
-	name = "None";	
-}
-
-//Makes the Job Class with parameters
-//Param: a->arrival, b->service,c->prioirty nam-> name of service
-Job::Job(int a, int b, int c, std::string nam = "None")
-{
-	arrivalTime = a;
-	serviceTime = b;
-	priority = c;
-	name = nam;
-	
-}
+////class for your basic Job; (copied from project2)
```

Comment:
This could have just been renamed something else rather than commented out, not really a compliant but just a suggestion.

Reply:
ya, excuse my sloppiness haha. I was in a testing spree and went with the quickest fix I could think of.